app/views/podcasts/show.html.erb

```diff
@@ -1,10 +1,105 @@
-<p style="color: green"><%= notice %></p>
+<% if notice %>
+  <p style="color: green"><%= notice %></p>
```

Comment:
You can probably drop this - we've been using toasts.  And I'm not sure one would ever show up on this page?